Yosemite/Yosemite/Stores/ProductStore.swift

```diff
@@ -539,18 +539,26 @@ private extension ProductStore {
 
     func generateProductSharingMessage(siteID: Int64,
                                        url: String,
+                                       name: String,
+                                       description: String,
                                        languageCode: String,
                                        completion: @escaping (Result<String, Error>) -> Void) {
         let prompt = [
-            "Your task is to help a merchant create a message to share with their customers a product in the provided URL: \(url)",
-            "Some requirements for the message are:",
+            "Your task is to help a merchant create a message to share with their customers the below product.",
+            "Product name: \(name)",
+            "Product description: \(description)",
+            "Product URL: \(url)",
+            "Some requirements for the message are: ",
             "- In language \(languageCode)",
             "- The length should be up to 3 sentences",
             "- Try to make shorter sentences, using less difficult words to improve readability",
-            "- Add related hashtags at the end of the message."
+            "- Add related hashtags at the end of the message.",
+            "- Do not include the URL in the message.",
         ].joined(separator: "\n")
         Task {
-            let result = await Result { try await generativeContentRemote.generateText(siteID: siteID, base: prompt) }
+            let result = await Result { try await generativeContentRemote.generateText(siteID: siteID, base: prompt)
+                    .trimmingCharacters(in: CharacterSet(["\""]))  // Trims quotation mark
```

Comment:
Nit: I wonder if there's any edge case where quotation marks might be needed in the middle of sentences. Should we remove only the marks at the beginning and the end?

```suggestion
                    .removingPrefix("\"")
                    .removingSuffix("\"")
```

Reply:
- `trimmingCharacters` only removes the characters in the `CharacterSet` from the beginning and end of the String. [Docs](https://developer.apple.com/documentation/foundation/nsstring/1415462-trimmingcharacters)
- We are also unit testing this behaviour [here](https://github.com/woocommerce/woocommerce-ios/pull/9921/files#diff-e646aaeb37f27f6e8e28316b94a04243eb7e834bf654550b4fcf2f825782dbe5R1876).

Kindly let me know if I am missing something here.

Reply:
Merging the PR now. I will make changes in a separate PR if needed.

Reply:
My bad, please ignore my comment then.

Reply:
No problem. Thanks again for the review! 🙇